schema/Readme.md

```diff
@@ -688,12 +690,22 @@ Specifies a default value if null is provided
 
 Disambiguates relationships when needed
 
-- name: _(optional)_ defines the name of the relationship
+###### Arguments
+
+- name: _(optional, except when required for disambiguation)_ defines the name of the relationship. The name needs to be disambiguated in the following cases:
+  - There is more than one relation between two types.
+  - The relation is a self-relation with two fields.
```

Comment:
I think we can condense this to one rule: A relation is considered ambiguous if a model has 2 fields that refer to the same model. This covers normal and self relations imo.